R/moma_sfpca.R

```diff
@@ -50,10 +6,70 @@ SFPCA <- R6::R6Class("SFPCA",
             # internal functions
             private$check_input_index <- FALSE
             res <- self$get_mat_by_index(
-                alpha_u, alpha_v, lambda_u, lambda_v
+                alpha_u = alpha_u,
+                alpha_v = alpha_v,
+                lambda_u = lambda_u,
+                lambda_v = lambda_v
+            )
+            private$check_input_index <- TRUE
+            return(res)
+        },
+        private_left_project = function(newX, ...,
+                                                alpha_u = 1, alpha_v = 1, lambda_u = 1, lambda_v = 1, rank = 1) {
```

Comment:
Is this really formatted correctly? It looks super weird.

Reply:
`rstyler` did process the file.

Reply:
I didn't find any useful info on customizing this type of alignment.

Strangely here (https://github.com/DataSlingers/MoMA/pull/46#discussion_r315396442) has the correct indentation.